### PR TITLE
Use fix multiprocessing bug on macOS where `qsize()` is not implemented

### PR DESCRIPTION
**Pull request by Donnype**
### Changes
This should fix the MacOSX issues due to `qsize()` not working (as documented in the library.. For the last 15 years..)

### Proof
Tested on my Mac, we could consider adding a matrix in github but this is probably overkill.

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [x] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by praseodym — DISMISSED** [review]
lgtm

File: boefjes/boefjes/app.py [hunk 1]
```diff
@@ -27,26 +27,27 @@ class SchedulerWorkerManager(WorkerManager):
...
-        self.client_factory = client_factory
-        self.scheduler_client = client_factory()
+        self.scheduler_client = scheduler_client
         self.settings = settings
 
-        self.task_queue = mp.Queue()
-        self.handling_tasks = mp.Manager().dict()
+        manager = mp.Manager()
+
+        self.task_queue = manager.Queue()  # multiprocessing.Queue() will not work on OSX, see mp.Queue.qsize()
```
Comment (praseodym) [inline boefjes/boefjes/app.py:40]: Renamed to macOS since then :)

**Review by ammar92 — APPROVED** [review]

**Comment by underdarknl** [comment]
@noamblitz could you see if the problem persists?

**Review by noamblitz — APPROVED** [review]
Works, nice fix.

**Review by underdarknl — APPROVED** [review]

**Pull request merged**